bin/resources/GameIndex.yaml

```diff
@@ -35833,7 +35833,7 @@ SLPS-25934:
   name: "Samurai Spirits - Rokuban Shoubu [NeoGeo Online Collection the Best]"
   region: "NTSC-J"
 SLPS-25935:
-  name: "The King of Fighters '98 Ultimate Match (NeoGeo Online Collection the Best)"
+  name: "King of Fighters '98 Ultimate Match (NeoGeo Online Collection the Best), The"
```

Comment:
```suggestion
  name: "King of Fighters '98 Ultimate Match, The (NeoGeo Online Collection the Best)"
```

Reply:
I will say however how these are done feels somewhat inconsistent, but there's no clear solution, but the The should certainly come before the packaging type.